python3/README.md

```diff
@@ -0,0 +1,10 @@
+# Python3
+
+Before running this test code, make sure you have your `api_key` set as the environment variable, `FORWARD_FINANCING_API_KEY`.
+
+Also, ensure that you have [requests](http://docs.python-requests.org/en/master/) installed as that is how we will be making web requests.
+
+### Running the Script
+- `cd` into this directory (`/python3`)
+- change `"reference_id": "unique_string_here"` to some unique string
```

Comment:
Where do they get that unique string from?

Reply:
It's any unique identifier string that they want to provide. The only requirement is that that string has never been used to submit a lead with on our API.

Reply:
Thanks! Maybe we can suggest that it could be their own identification for the lead or an uuid of some sort that they can keep for their record? I am not sure and I am not particularly too fussed about this. It's an open question.

Reply:
Got it! Thanks!